lib/internal/http2/util.js

```diff
@@ -415,16 +417,18 @@ function mapToHeaders(map,
       if (isArray) {
         for (var k = 0; k < value.length; k++) {
           val = String(value[k]);
-          ret.push([key, val]);
+          ret += `${key}\0${val}\0`;
         }
+        count += value.length;
       } else {
         val = String(value);
-        ret.push([key, val]);
+        ret += `${key}\0${val}\0`;
+        count++;
       }
     }
   }
 
-  return ret;
+  return [ ret, count ];
```

Comment:
I know, but as far as I know the house style is still `[a, b, c]`. To be honest, I personally prefer `[ a, b, c ]` over `[a, b, c]`, but enforcing spacing within arrays would cause over 7400 errors within our code. But as the majority seems to prefer arrays without spaces anyway, I would go for consistency, even though @refack warned me that a PR *might* not be too successful, depending on the churn.